projects/npm-tools/packages/npm-scripts/src/scripts/createBridges.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {addNamespace} = require('@liferay/js-toolkit-core');
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (bridges, dir) {
+	const projectPackageJson = JSON.parse(
+		fs.readFileSync('package.json', 'utf-8')
+	);
+
+	Object.entries(bridges).forEach(([packageName, exportName]) => {
+		const packageJson = JSON.parse(
+			fs.readFileSync(
+				require.resolve(`${packageName}/package.json`),
+				'utf-8'
+			)
+		);
+
+		const namespacedVersionedPackageName = addNamespace(
+			`${packageJson.name}@${packageJson.version}`,
+			projectPackageJson
+		);
+
+		const packageDir = path.join(
+			dir,
+			'node_modules',
+			namespacedVersionedPackageName
+		);
+
+		fs.mkdirSync(packageDir, {recursive: true});
+
+		fs.writeFileSync(
+			path.join(packageDir, 'package.json'),
+			JSON.stringify(
+				{
+					...packageJson,
+					name: addNamespace(packageJson.name, projectPackageJson),
+				},
+				null,
+				'\t'
+			)
+		);
+
+		let {main: moduleFileName} = packageJson;
+
+		moduleFileName = moduleFileName || 'index.js';
+		moduleFileName = moduleFileName.replace(/^\.\//, '');
+
+		if (!moduleFileName.toLowerCase().endsWith('.js')) {
+			moduleFileName += '.js';
+		}
+
+		fs.mkdirSync(path.join(packageDir, path.dirname(moduleFileName)), {
+			recursive: true,
+		});
+
+		const moduleName = moduleFileName.replace(/\.js$/i, '');
+
+		fs.writeFileSync(
+			path.join(packageDir, moduleFileName),
+			`
+(function() {
+	const getModule = window[Symbol.for('__LIFERAY_WEBPACK_GET_MODULE__')];
+
+	getModule('${projectPackageJson.name}').then(
+		function({${exportName}}) {
+			Liferay.Loader.define(
+				"${namespacedVersionedPackageName}/${moduleName}", ['module'],
```

Comment:
Double and single quotes on this line — probably better to just stick with single.